cmd/commands/import.go

```diff
@@ -130,6 +130,7 @@ func bulkImportCmdF(command *cobra.Command, args []string) error {
 		if lineNumber != 0 {
 			cmd.CommandPrettyPrintln(fmt.Sprintf("Error occurred on data file line %v", lineNumber))
 		}
+		return err
```

Comment:
Without this, the bulkload command would not fail execution, and would otherwise appear successful to the shell and loadtest invocation of same.